common/infrastructure/Cargo.toml

```diff
@@ -23,7 +23,7 @@ mime = { workspace = true }
 openssl = { workspace = true }
 opentelemetry = { workspace = true }
 opentelemetry-otlp = { workspace = true }
-opentelemetry_sdk = { workspace = true, features = ["rt-tokio"] }
+opentelemetry_sdk = { workspace = true, features = ["rt-tokio-current-thread"] }
```

Comment:
> rt-tokio-current-thread: Spawn telemetry tasks on a separate runtime so that the main runtime won’t be blocked. [link 1](https://docs.rs/opentelemetry_sdk/latest/opentelemetry_sdk/#crate-feature-flags), [link 2](https://github.com/OutThereLabs/actix-web-opentelemetry?tab=readme-ov-file#exporter-configuration)